internal/runners/shell/shell.go

```diff
@@ -58,12 +59,25 @@ func New(prime primeable) *Shell {
 func (u *Shell) Run(params *Params) error {
 	logging.Debug("Shell %v", params.Namespace)
 
-	proj, err := runbitsProject.FromNamespaceLocal(params.Namespace, u.config, u.prompt)
-	if err != nil {
-		if runbitsProject.IsLocalProjectDoesNotExistError(err) {
-			return locale.WrapInputError(err, "err_shell_project_does_not_exist", "Local project does not exist.")
+	var proj *project.Project
+	var err error
+	if params.Namespace.Owner != "" || params.Namespace.Project != "" {
```

Comment:
```suggestion
	if params.Namespace.Project != "" {
```

The owner on its own doesn't seem like it would work.